### ISSUE TITLE
CNN with the same will run twice for value and policy networks.

### ISSUE THREAD

**Issue by liyuanpeng01** [idlrl/flare#38]
https://github.com/idlrl/flare/blob/36a34dc122dc818eb16107981fb4586d1b80b04e/flare/algorithm_zoo/simple_algorithms.py#L35

https://github.com/idlrl/flare/blob/36a34dc122dc818eb16107981fb4586d1b80b04e/flare/algorithm_zoo/simple_algorithms.py#L50

**Comment by skylian** [comment]
Yes. This is a known issue. The reason we keep it that way is, as a general algorithm implementation, we don't know whether the value network and the policy network has a shared part or not.
